scripts/downstream_checks.py

```diff
@@ -36,12 +35,15 @@
 r = resp.json()
 
 for e in r['data']['search']['edges']:
-    if e.get('node', {}).get('title', '') != f'Upgrade pulumi-terraform-bridge to {c}':
+    if not str(e.get('node', {}).get('title', '')).endswith(
+        f'Upgrade pulumi-terraform-bridge to {c}'
+      ):
         continue
 
     checks = e.get('node', {}).get('commits', {}).get('nodes', [{}])[0]['commit']['statusCheckRollup']['state']
     if checks == 'SUCCESS':
         url = e['node']['url']
+        print('CLOSING', e['node']['url'])
```

Comment:
```suggestion
        url = e['node']['url']
        print(f'CLOSING {url}')
```

Reply:
I've matched the style of the script - I'll leave refactoring for another PR, since I'll prob need this for https://github.com/pulumi/pulumi-terraform-bridge/issues/1721

Reply:
well, just noticed the repeated expression 🤷 , will address in another PR